src/gui/ObsListCreateEditDialog.hpp

```diff
@@ -31,117 +31,118 @@
 class Ui_obsListCreateEditDialogForm;
 
 class ObsListCreateEditDialog : public StelDialog {
-Q_OBJECT
+	Q_OBJECT
 
 public:
-    static auto Instance(std::string listUuid) -> ObsListCreateEditDialog *;
+	static ObsListCreateEditDialog * Instance(std::string listUuid);
```

Comment:
If you put that you have this warning:
"Clang-Tidy: Use a trailing return type for this function". This why I put this code:
static auto Instance(std::string listUuid) -> ObsListCreateEditDialog *;

In my code I tried to correct a lot of Clang-Tidy warning.

Reply:
QtCreator also has clang-tidy and clazy warning systems. This helped a lot getting rid of many issues, suggesting trailing return types being none of them. Probably it can be fine-tuned somehow on your system. I really dislike the new c++ syntax with `auto function(args) -> type` which does IMHO nothing to increase readability. Esp. starting to add this after 20 years of project history.

Reply:
@gzotti 
"I really dislike the new c++ syntax with auto function(args) -> type which does IMHO nothing to increase readability" -> I don't like too ;)

Reply:
So we will not use this syntax for now.